### DISCUSSION TITLE
在英文模板中，修改subsection的配置不起效

### DISCUSSION THREAD

**Discussion by htyqaz** [sjtug/SJTUThesis#806] [提问] [answered]
在sjtuthesis.cls中修改各级标题设置，只有subsection级的设置一直不起作用，其他级可以正常更改字体。
设置代码如图：
![image](https://user-images.githubusercontent.com/66247474/168596085-862d2fdd-cbba-444b-baf5-18c2bf1cc97c.png)
编译结果如图：
![image](https://user-images.githubusercontent.com/66247474/168596150-bfb45a30-ffef-4d84-a568-08d9b286c260.png)
可见只有三级标题未改变字号且加粗。

**Comment by skyzh** [comment, marked as answer]
本科三级标题有另外设置：

https://github.com/sjtug/SJTUTeX/blob/1816662df3c1198f8b6315bbd3a86b9e3df0b6bb/source/sjtuthesis.dtx#L2503-L2508

还需要修改 [sjtuthesis-undergraduate.ltx](https://github.com/sjtug/SJTUThesis/blob/master/texmf/tex/latex/sjtuthesis/sjtuthesis-undergraduate.ltx)。

**Reply by skyzh** [reply]
如果不想从 dtx 从头编译，可以修改：

https://github.com/sjtug/SJTUThesis/blob/b164d0f5b3087ed86c1038b5a85014b47a92c6f2/texmf/tex/latex/sjtuthesis/sjtuthesis-undergraduate.ltx#L59-L61
